src/components/HistoryTables/MessageHistory/MessageConfirmedRow.tsx

```diff
@@ -48,10 +48,11 @@ export default function MessageHistoryRow(props: MessageHistoryRowProps) {
       <TD>{message.height}</TD>
       <TD>{age}</TD>
       <TD>
-        <AddressWOptionalLink
-          address={message.from?.robust || message.from.id}
-          addressHref={addressHref}
-          inspectingAddress={inspectingAddress}
+        <AddressLink
+          id={message.from.robust ? '' : message.from.id}
+          address={message.from.robust}
+          disableLink={!incoming}
```

Comment:
`disableLink` should be disabled when the `inspectingAddress` _is_ the ID or robust address.

So something like:

```
disableLink={convertAddrPrefix(message.from.robust) === convertAddrPrefix(inspectingAddress) || convertAddrPrefix(message.from.id) === convertAddrPrefix(inspectingAddress)}
```

Reply:
Isn't that the same as what `incoming` already checks? 

```
const incoming = useMemo(
  () =>
    message.to.robust === inspectingAddress ||
    message.to.id === inspectingAddress,
  [message.to, inspectingAddress]
)
```

Also, is `convertAddrPrefix` necessary and do you mean `converAddrToFPrefix` that exists in some app repos?
It seems to just slice the first character and replace with an `f`.

Reply:
ooh wait, incoming checks it for the `to` address, I see